src/lib/components/EventScheduleModal/index.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BaseModalElement} from '../BaseModalElement';
+import {html} from 'lit-element';
+import './_styles.scss';
+import {generateIdSalt} from '../../utils/generate-salt';
+
+/**
+ * @fileoverview Modal element for session information.
+ */
+
+class EventScheduleModal extends BaseModalElement {
+  static get properties() {
+    return {
+      sessionRow: Element,
+      _sessionName: String,
+      _authorsPart: Element,
+      _abstractPart: Element,
+      _titleId: String,
```

Comment:
Should these be written as?:

```
_sessionName: { type: String }
```

https://lit-element.polymer-project.org/guide/properties#conversion-type

Does lit support the Element type? I thought it could only do Object
https://lit-element.polymer-project.org/guide/properties#conversion-type

Reply:
Apparently I forgot how to write lit. Using `Element` looks pretty but I'm pretty sure triggers default behavior. Turns out the type only helps you when you're converting from property to attribute, and we're not doing this here, so...

I've fixed this up and changed everything to `Object`.